app/planet_routes.py

```diff
@@ -0,0 +1,52 @@
+from flask import Blueprint, jsonify, abort, make_response
```

Comment:
Consider moving this file into a `routes` folder. This is useful even when we don't have another set of routes to store there, since we'll be adding additional files, and having things grouped logically can help us find our way around our code.